src/test/java/core/basesyntax/service/impl/FileWriterImplTest.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.service.impl;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class FileWriterImplTest {
+    private static final String PATH_TO_TEST_OUTPUT_FILE = "src\\test\\resources\\testOutput.csv";
+    private FileWriterImpl fileWriter;
+
+    @BeforeEach
+    void setUp() {
+        fileWriter = new FileWriterImpl();
+    }
+
+    @Test
+    void writeToFile_wrongPathToFile_notOk() {
+        Assertions.assertThrows(

```

Comment:
Please move `Assertions.assertThrows` to static import and make code more concise and readable. 
Fix all similar cases for all assertions in all test classes.